Sources/Networking/HTTPClient/URLSessionConfigurationFactory.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  URLSessionConfigurationFactory.swift
+//
+//  Created by Andrés Boedo on 5/29/23.
+
+import Foundation
+
+protocol URLSessionConfigurationFactoryType {
+    func urlSessionConfiguration(requestTimeout: TimeInterval) -> URLSessionConfiguration
```

Comment:
I don't care that much, but we should decide whether we want to keep the consistency in the style guide of spaces around types.